Sources/Fuzzilli/Modules/NetworkSync.swift

```diff
@@ -78,7 +78,7 @@ protocol MessageHandler {
 /// A connection to a network peer that speaks the above protocol.
 class Connection {
     /// File descriptor of the socket.
```

Comment:
Nit: maybe update this comment as well and state that it'll be a socket file descriptor on POSIX and a HANDLE on Windows? Then the code in lines 112-116 is also clear